[PL-52031] Update PL Subscriptions content (#7186)
harness/developer-hub: docs/platform/get-started/subscriptions-licenses/subscriptions.md

import Manage from '/docs/platform/shared/subscription-manage.md';
import View from '/docs/platform/shared/subscription-view-subscriptions.md';

> You have the capability to view and manage module subscription details as well as license information directly within your Harness account. This includes monitoring current subscriptions, reviewing license allocations, and adjusting as necessary to meet your organizational needs.

Author: ```suggestion
You can view and manage module subscription details as well as license information directly within your Harness account. This includes monitoring current subscriptions, reviewing license allocations, and adjusting as necessary to meet your organizational needs.
```

## Developer 360 subscriptions

You can subscribe to Harness modules using the [Developer 360](https://www.harness.io/pricing) subscription model. The Developer 360 subscription offers three plans: **Free**, **Startup**, and **Enterprise**. All plans include [Harness AIDA](/docs/category/harness-aida), our AI Development Assistant.

:::info
This model spans all modules except Cloud Cost Management, which is on a *Cloud Spend* model.
:::

The following sections explain how this subscription model works across the various modules on the platform.

### Who is a developer?

> A **Developer** is each person who contributes to code development, deployment, security, maintenance, optimization or any other technical activities related to software code, that is managed with Harness.

Reviewer: ```suggestion
A **Developer**  contributes to code development, deployment, security, maintenance, optimization, or all other technical activities related to software code that is managed with Harness.
```

These users leverage Harness to create code repositories, build, test, secure, and deploy software, conduct chaos experiments, manage SLOs, handle feature flags, and more.

The definition of a **Developer** (and any associated consumption entitlements) remains consistent across all plans: **Free**, **Startup**, and **Enterprise**.

### Modules with direct developer tracking

> The following Harness modules are integral to a **Developer's** daily workflow, each requiring a developer license for every user involved.

Reviewer: ```suggestion
The following Harness modules are integral to a **Developer's** daily workflow. Each module requires a separate developer license for every user involved.
```

- **Code Repository (CODE):** All users on the Harness Platform who have permission to access Harness CODE module are considered as **CODE Developers** and consume a license.
- **Continuous Integration (CI):** All users on the Harness Platform who have permission to access Harness CI module are considered as **CI Developers** and consume a license.
- **Internal Developer Portal (IDP):** All users on the Harness Platform who have permission to access Harness IDP module are considered as **IDP Developers** and consume a license.
- **Feature Flags (FF):** All users on the Harness Platform who have permission to create / update / delete Feature Flags and/or Delivery Groups, within the Harness FF module, are considered as **FF Developers** and consume a license.

> - **Software Engineering Insights (SEI):** All users read from a configured SCM (Source Code Management) tool are considered as **SEI Developers** and consume a license.

Reviewer: ```suggestion
- **Software Engineering Insights (SEI):** All users who read from a configured SCM (Source Code Management) tool are considered as **SEI Developers** and consume a license.
```
Not sure if I'm changing the meaning here....

Harness Feature Flags is a feature flag rollout and management module that tracks **Developers** and **MAU** (Monthly Active Users) for license consumption.

> All users on the Harness Platform who have permission to create,update, or delete Feature Flags and/or Delivery Groups, within the Harness FF module, are considered as **FF Developers** and consume a license.

Reviewer: ```suggestion
All users on the Harness Platform who have permission to create, update, or delete Feature Flags and/or Delivery Groups, within the Harness FF module, are considered **FF Developers** and consume a license.
```

<details>
<summary>Chaos Engineering (CE) Services</summary>

> Chaos Engineering (CE) facilitates the execution of chaos experiments across diverse services within an R&D organization, aimed at enhancing their resilience. A **Service** represents an autonomous software unit managed and tracked through Harness CD and GitOps. This typically maps to:

Reviewer: ```suggestion
Chaos Engineering (CE) makes it easy to run chaos experiments across diverse services within an R&D organization, aimed at enhancing their resilience. A **Service** represents an autonomous software unit managed and tracked through Harness CD and GitOps. This typically maps to:
```

</details>

<details>
<summary>Security Testing Orchestration (STO) Security Scans</summary>

> A **Security Scan** is defined as the execution of the Security Testing Orchestration (STO) step within a pipeline. This involves scanning an artifact (referred to as the **Target**), which can be a Repository, Docker image, or a live application, for security vulnerabilities.

Reviewer: ```suggestion
A **Security Scan** is defined as the execution of the Security Testing Orchestration (STO) step within a pipeline. This involves scanning a **Target** for security vulnerabilities. A target can be a repository, container image, configuration, or live application.
```

This means every developer license for a module includes a corresponding consumption entitlement, where applicable.

> Additionally, you always have the flexibility to acquire additional consumption entitlements as needed through add-on purchases.

Reviewer: ```suggestion
You can always acquire additional consumption entitlements as needed through add-on purchases.
```
